aksel.nav.no/website/pages/templates/soknad-introside-alderspensjon/soknad-introside-alderspensjon.tsx

```diff
@@ -0,0 +1,363 @@
+                    </BodyLong>
+                  </Accordion.Content>
+                </Accordion.Item>
+                <Accordion.Item>
+                  <Accordion.Header>Vi lagrer svar underveis</Accordion.Header>
+                  <Accordion.Content>
+                    <BodyLong>
+                      Her skal det stå informasjon om hvordan denne søknaden
+                      mellomlagrer informasjonen til søkeren og hvor lenge
+                      informasjonen lagres. Vi skal informere om mellomlagring
+                      ved både automatisk lagring og ved samtykke til lagring
+                      med lagre-knapp.
+                    </BodyLong>
+                  </Accordion.Content>
+                </Accordion.Item>
+              </Accordion>
+            </Box>
+            <div>
+              <BodyLong>
+                Det er viktig at du gir oss riktige opplysninger slik at vi kan
+                behandle saken din.{" "}
+                <Link href="#gi-riktige-opplysninger">
+                  Les mer om viktigheten av å gi riktige opplysninger.
+                </Link>
+              </BodyLong>
+              <Box paddingBlock="4 8">
+                <Checkbox>
+                  Jeg vil svare så godt jeg kan på spørsmålene i søknaden.
+                </Checkbox>
+              </Box>
+              <Button
+                variant="primary"
+                icon={<ArrowRightIcon aria-hidden />}
+                iconPosition="right"
+              >
+                Start søknad
+              </Button>
+            </div>
+          </VStack>
+        </Page.Block>
+      </VStack>
+      <Env />
+    </Page>
+  );
+}
+
+function Header() {
+  return <div id="decorator-header" />;
+}
+
+function Footer() {
+  return <div id="decorator-footer" />;
+}
+
+const MILJO_URL = "https://www.nav.no/dekoratoren";
+
+function Env() {
+  return (
+    <div
+      id="decorator-env"
+      data-src={`${MILJO_URL}/env?context=privatperson&simple=true&availableLanguages=[{"locale":"nb","url":"https://www.nav.no/person/kontakt-oss"},{"locale":"en","url":"https://www.nav.no/person/kontakt-oss/en/"}]`}
+    />
+  );
+}
+
+/**
+ * OBS: Dette er ikke anbefalt metode for å laste dekoratør!
+ * Se `nav-dekoratoren`-dokumentasjon for riktig implementasjon
+ * @see https://github.com/navikt/nav-dekoratoren
+ */
+function useDekorator() {
+  useEffect(() => {
+    const script = document.createElement("script");
+    script.src = `${MILJO_URL}/client.js`;
+    script.async = true;
+    document.body.appendChild(script);
+
+    const styles = document.createElement("link");
+    styles.href = `${MILJO_URL}/css/client.css`;
+    styles.rel = "stylesheet";
+    document.head.appendChild(styles);
+
+    return () => {
+      document.body.removeChild(script);
+      document.head.removeChild(styles);
+    };
+  }, []);
+}
+
+const ApplicationPictogram = ({ title, ...props }) => (
```

Comment:
Does it need to accept props? It has aria-hidden, so would think title wasn't necessary.

Reply:
No, you are right. 👍